common/djangoapps/course_modes/views.py

```diff
@@ -185,6 +187,12 @@ def get(self, request, course_id, error=None):
                 context["sku"] = verified_mode.sku
                 context["bulk_sku"] = verified_mode.bulk_sku
 
+        context['currency_data'] = []
+        if waffle.switch_is_active('local_currency'):
```

Comment:
Nit: Would it be slightly more readable to combine the two `if`s into one?

```python
if waffle.switch_is_active('local_currency') and 'edx-price-l10n' not in request.COOKIES
```

Reply:
i'd like to differentiate this feature being enabled, versus what the behavior is if it is enabled